tests/functionaltests/MainViewController.mm

```diff
@@ -27,9 +27,9 @@ @interface MainViewController ()
 @implementation MainViewController
 
 - (void)viewDidLoad {
-    self.view.backgroundColor = [UIColor lightGrayColor];
-
     [super viewDidLoad];
+
+    self.view.backgroundColor = [UIColor purpleColor];
```

Comment:
sounds fine, but curious about the reordering of calls?  that seems like unnecessary churn, or was it required for some reason?

---
In reply to: [97384555](https://github.com/Microsoft/WinObjC/pull/1771#discussion_r97384555) [](ancestors = 97384555)